syntax = "proto3";

service OrdererByRaft {
  rpc GetBlock(Transaction) returns(Block) {}
}

message Transaction {
  int32 sender = 1;
  int32 receiver = 2;
  float size = 3;
  float send_timestamp = 4;
}

message Block {
  int32 sender = 1;
  string receivers = 2; // separated by ","
  int32 id = 3;
  float size = 4;
  float send_timestamp = 5;
}